index.html

```diff
@@ -7,9 +7,27 @@
     <link rel="stylesheet" href="styles.css">
     <title>Learn code!</title>
 </head>
+<style>
+    @import url('https://fonts.googleapis.com/css2?family=Poppins:wght@500;700&display=swap');
+ </style>
 <body>
-    <h1>I am ready to start coding! :)</h1>
-    <button onclick="myFunction()">Try me</button>
+    <div class="container">
+        <div class="columnLeft">
+            <h1>Learn to code by<br>
```

Comment:
Hi! I would suggest to change the code like this:
```suggestion
            <h1>Learn to code by<br>watching others </h1>
```